src/drivers/uart_esc/uart_esc.cpp

```diff
@@ -523,6 +531,11 @@ int uart_esc_main(int argc, char *argv[])
 		}
 	}
 
+	// gets the parameters for the esc's pwm
+	param_get(param_find("PWM_UART_ESC_DISARMED"), &uart_esc::_pwm_disarmed);
```

Comment:
It got truncated in QGC but didn't fail on me or anything. Is that supposed to fail silently? I'll use the params in sensors anyway though.